src/lib/components/Toggle.svelte

```diff
@@ -7,13 +7,14 @@
   export let checked: boolean;
   export let ariaLabel: string;
   export let disabled = false;
+  export let testId = "toggle";
 
   const dispatch = createEventDispatcher();
 
   const id = nextElementId("toggle-");
 </script>
 
-<div class="toggle" class:disabled>
+<div class="toggle" class:disabled data-tid={testId}>
```

Comment:
Maybe update or add a test that uses it in `Toggle.spec.ts`?

Reply:
Done!

Reply:
Why not

![image](https://github.com/user-attachments/assets/96806ef2-0625-4273-bc48-02d90f79cbf5)